Xamarin.Forms.Core/BoxView.cs

```diff
@@ -8,6 +9,8 @@ public class BoxView : View, IElementConfiguration<BoxView>
 	{
 		public static readonly BindableProperty ColorProperty = BindableProperty.Create("Color", typeof(Color), typeof(BoxView), Color.Default);
 
+		public static readonly BindableProperty CornerRadiusProperty = BindableProperty.Create("CornerRadius", typeof(CornerRadius), typeof(BoxView), default(CornerRadius));
```

Comment:
use `nameof(CornerRadius)`

Reply:
Changed!. Right now there are property definitions using string name and others with nameof. It would be good to review them all (maybe in a PR?).

Reply:
we don't do massive refactoring. that cause major merges issues with unrelated branches.